readme.md

```diff
@@ -26,7 +36,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+ A post request is made using the jQuery method $.ajax and using the response data that it retrieves and converts to a JSON object to post on the client-side to the API. 
 ```
```

Comment:
Also looking for comments on how we go about (1) trigger the AJAX call in the first place (e.g., event listener) and (2) pulling information from the form (e.g., `$( "input" ).val( );`).